Develop

wathigo opened the pull request.

wathigo committed:
README file modified, initial commit

wathigo committed:
set-up files implemented

wathigo committed:
Merge pull request #1 from wathigo/set-up

set-up files implemented

wathigo committed:
SECTIONS IMPLEMENTATION
Implementation of three sections which include:
1. Navigation bar.
2. Video section.
3. Sidebar(to display images).

wathigo committed:
Merge pull request #2 from wathigo/implement-sections

SECTIONS IMPLEMENTATION

sonegillis committed:
POSITION SECTIONS
use the grid display to position the 3 sections
style the navbar

wathigo committed:
Merge pull request #3 from wathigo/position-sections

POSITION SECTIONS

wathigo committed:
EMBEDDING YOUTUBE VIDEO
1. Add the embed code for a youtube video in the html page.
2. Modify the css rules fpr a better appearance.

wathigo committed:
Merge pull request #4 from wathigo/embed-video

EMBEDDING YOUTUBE VIDEO

wathigo committed:
SIDEBAR IMPLEMENTATION
1. Image element fo the sidebar implemented.
2. h3 element for the heading implemented.
3. p element to show views and channel name implemented.
4. Grid layout used to position the elements.

wathigo committed:
Merge pull request #5 from wathigo/implement-sidebar

SIDEBAR IMPLEMENTATION

sonegillis committed:
Add the video description

wathigo committed:
Merge pull request #6 from wathigo/implement-video-description

Add the video description

mcrd25 commented on index.html:
```diff
@@ -0,0 +1,78 @@
+    <section class="video-section">
+      <iframe width="750" height="415"
+              src="https://www.youtube.com/embed/ZnuwB35GYMY"
+              frameborder="0" allow="accelerometer; autoplay;
```
Hi, guys the `frameborder` attribute is no longer supported by HTML. it may be better to not specify a frameborder, and use CSS to apply borders instead if you need it. 

Please address in all occurrences 😄 .

mcrd25 commented on index.html:
```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
```
Don't forget it's best practice to add `lang` attribute to `html` tag.

mcrd25 commented:
# Neat project! Simple and to the point! 👍 🥇 
## Please address review comments 😄 

This project is about testing your ability to embed images and video and you have done this successfully. Please address review comments and you may move on. There are however two more things I'd need you to do first:
- Please set up stickler for this project and see this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [this](https://github.com/microverseinc/linters-config/tree/master/css) link.
- Also make README a bit more descriptive by adding the title of the project, description (maybe even link Odin project link if you want to), have a link to live version of the project (when that time reaches) and add your GitHub usernames and link them (can do same with other similar social media). Click [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) for some markdown syntax.

## Suggestion
- Maybe make sidebar videos (preview videos/up next section) hoverable, i.e have `cursor: pointer` effect.

**Great job though! Met all project requirements!** 👍  💯 🥇 **. Almost ready for** 🚀 .

stickler-ci committed:
Adding .stickler.yml

wathigo committed:
Merge pull request #9 from wathigo/add-stickler-config

Adding .stickler.yml

wathigo committed:
FEEDBACK IMPLEMENTATION
1. Removed the frameborder attribute from the iframe element.
2. Added a lang attribute to the html element.
3. Added a hover effect on the sidebar items.

wathigo committed:
Merge pull request #10 from wathigo/fix-issue

FEEDBACK IMPLEMENTATION

wathigo committed:
Update README.md

sonegillis committed:
delete empty css rules

wathigo committed:
Merge pull request #11 from wathigo/fix-linter-issues

delete empty css rules

sonegillis committed:
change grid-gap to gap in styles.css

wathigo committed:
Merge pull request #12 from wathigo/fix-linter-issues

change grid-gap to gap in styles.css

mcrd25 commented on css/styles.css:
```diff
@@ -0,0 +1,41 @@
+  width: 40%;
+  margin: 0 auto;
+}
+#search {
```
You can ignore the id as selector error.

mcrd25 reviewed (approved):
# You may move on. 🚢  it to master and 🚀 .
You accomplished the basics of the project which what was needed! Good job!
Would say more detail can be put into this one in the future though to make it like 💯.
No need for further review! 
**Congratulations on completing your first project at Microverse!!!** 🎊 🎉 

** I let the `gap` issue be, but it is not fully supported by all browsers yet so for now `grid-gap` may be the safer bet. **

wathigo commented:
Thank you @mcrd25
